scripts/convert_merged_tract_to_dpdd.py

```diff
@@ -111,6 +182,9 @@ def write_dataframe_to_files(df,
 The output files will have columns those specified in the LSST DPDD
 (https://ls.st/dpdd), plus 'tract' and 'patch' for convenience.
 
+The input filename is expected to match 'trim_merged_tract_.*\.hdf5$'
+(all tracts) or 'trim_merged_tract_{:04d}\.hdf5$' (one tract).
+
```

Comment:
Agree - as long as it is referred as to in the relevant routines (plus link to your review comment on this).

I mentioned it because while doing tests, it was not obvious to me which input catalog I needed to use. But other users might be more aware.